src/app/main/login/login.component.ts

```diff
@@ -26,13 +28,21 @@ export class LoginComponent implements OnInit {
     ngOnInit() {
     }
 
-    refresh(status: boolean) {
+    refresh() {
+        // check if a session is active
+        if (this._session.getSession()) {
```

Comment:
could be `if(status) {`, because you get the status from login-form

Reply:
refactored in 34e4acf